src/main/kotlin/org/rust/lang/core/psi/ext/RsMacroCall.kt

```diff
@@ -44,7 +44,7 @@ abstract class RsMacroCallImplMixin : RsStubbedElementImpl<RsMacroCallStub>,
 }
 
 val RsMacroCall.macroName: String
-    get() = path.referenceName
+    get() = path.referenceName.orEmpty()
```

Comment:
It seems we want to return null if `referenceName` is null. Let's Add the corresponding `TODO` here.
I don't want to fix it in this PR - it's already quite large